ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-access-lists.j2

```diff
@@ -0,0 +1,16 @@
+{% if ip_access_lists is arista.avd.defined %}
+
+## IP Access-lists
+{%     if ip_access_lists_max_entries is arista.avd.defined %}
+
+### IP Access-lists Summary
+
+> The maximum number of ACL entries allowed to be provisioned per switch: {{ ip_access_lists_max_entries }}
```

Comment:
Sorry for not catching this earlier. This should be a `-` in the beginning instead of `>`